arm-storage/2017-06-01/swagger/storage.json

```diff
@@ -0,0 +1,1395 @@
+          "keyName": {
+            "type": "string"
+          },
+          "keyVersion": {
+            "type": "string"
+          },
+          "keyVaultUri": {
+            "type": "string"
+          }
+      },
+      "description": "Properties of key vault."
+    },
+    "Encryption": {
+      "properties": {
+        "services": {
+          "$ref": "#/definitions/EncryptionServices",
+          "description": "List of services which support encryption."
+        },
+        "keySource": {
+          "type": "string",
+          "description": "The encryption keySource (provider). Possible values (case-insensitive):  Microsoft.Storage, Microsoft.KeyVault",
+          "enum": [
+            "Microsoft.Storage",
+            "Microsoft.KeyVault"
+          ]
+        },
+        "keyVaultProperties":{
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Properties provided by key vault."
+        }
+      },
+      "required": [ "keySource" ],
+      "description": "The encryption settings on the storage account."
+    },
+    "StorageAccountPropertiesCreateParameters": {
+      "properties": {
+        "customDomain": {
+          "$ref": "#/definitions/CustomDomain",
+          "description": "User domain assigned to the storage account. Name is the CNAME source. Only one custom domain is supported per storage account at this time. To clear the existing custom domain, use an empty string for the custom domain name property."
+        },
+        "encryption": {
+          "$ref": "#/definitions/Encryption",
+          "description": "Provides the encryption settings on the account. If left unspecified the account encryption settings will remain the same. The default setting is unencrypted."
+        },
+        "accessTier": {
+          "type": "string",
+          "description": "Required for storage accounts where kind = BlobStorage. The access tier used for billing.",
+          "enum": [
+            "Hot",
+            "Cool"
+          ],
+          "x-ms-enum": {
+            "name": "AccessTier",
+            "modelAsString": false
+          }
+        },
+        "supportsHttpsTrafficOnly": {
+          "type": "boolean",
+          "default": "false",
+          "x-ms-client-name": "EnableHttpsTrafficOnly",
+          "description": "Allows https traffic only to storage service if sets to true."
+        }
+      },
+      "description": "The parameters used to create the storage account."
+    },
+    "Identity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
+          "enum": [
+            "SystemAssigned"
```

Comment:
This is the only value we honored for identity currently. Just like the key source we are updating this time, which also used to be "enum" with only one value but we are adding another value.